index.html

```diff
@@ -14,6 +14,96 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+ <div class="container">
+<header>
+  <h1>Steven Jefferson</h1>
+
+  <nav>
+    <a href="#">Home</a>
+    <a href="#">About</a>
+    <a href="http://nevets615.github.io">Portfolio</a>
+    <a href="mailto:surj316@yahoo.com">Contact</a>
+  </nav>
+  
+</header>
+<div class="title">
+<h2>Web Developer</h2>
+
+<p class=address>
+<p>5111 Sourwood Rd</p> 
+<p>Durham NC</p> 
+<p>27510</p> 
+<p>(443) 562-5033</p>
+<a class="address" href="mailto:surj316@yahoo.com">surj316@yahoo.com</a>
```

Comment:
if these are you actual addresses (not sure if they're just placeholders), I'd recommend changing them to be fake since this is public on github, just in case